nostr-verify.php

```diff
@@ -31,8 +31,8 @@ function flush_rewrite_rules() {
 	generate_rewrite_rules();
 	\flush_rewrite_rules();
 }
-register_activation_hook( __FILE__, __NAMESPACE__ . '\flush_rewrite_rules' );
-register_deactivation_hook( __FILE__, '\flush_rewrite_rules' );
+register_activation_hook( __FILE__, __NAMESPACE__ . '\\flush_rewrite_rules' );
+register_deactivation_hook( __FILE__, 'flush_rewrite_rules' );
```

Comment:
I wouldn't remove this one, since this is aiming to trigger core's function:

```suggestion
register_deactivation_hook( __FILE__, '\flush_rewrite_rules' );
```

Reply:
When referencing a callable as a string, it is always fully qualified.
Please see https://3v4l.org/efBqE

Reply:
... and https://3v4l.org/RpZ1c

Reply:
Ah, perfect. 👍